llama_index/embeddings/base.py

```diff
@@ -78,7 +78,7 @@ def get_query_embedding(self, query: str) -> List[float]:
         query_tokens_count = len(self._tokenizer(query))
         self._total_tokens_used += query_tokens_count
         self.callback_manager.on_event_end(
-            CBEventType.EMBEDDING, payload={"num_nodes": 1}, event_id=event_id
+            CBEventType.EMBEDDING, payload={"chunks": [query]}, event_id=event_id
```

Comment:
Is this an intended change?

Reply:
curious why the rename?

Reply:
In order to track tokens, we need to log the entire list of text that was embedded. So `num_nodes` didn't quite make sense anymore. 99% sure none of the other callbacks were using this though